lms/djangoapps/courseware/url_helpers.py

```diff
@@ -9,19 +9,23 @@
 from xmodule.modulestore.django import modulestore
 
 
-def get_redirect_url(course_key, usage_key):
+def get_redirect_url(course_key, usage_key, unified_course_view=False):
```

Comment:
FYI, there's a helper method to get you the request object for the current thread that Nimisha told me about. It is ``RequestCache.get_current_request()`` and here's an example usage:

https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/tabs.py#L57